docs/guides/building-a-360-image-gallery.md

```diff
@@ -6,10 +6,10 @@ parent_section: guides
 order: 3
 examples:
   - title: 360&deg; Image Gallery
-    src: https://glitch.com/edit/#!/aframe-gallery?path=index.html
+    src: https://github.com/aframevr/aframe/tree/master/examples/showcase/360-gallery/index.html
 ---
 
-[glitch]: https://glitch.com/edit/#!/aframe-gallery?path=index.html
```

Comment:
I wish we could have both from both worlds for the docs. Easy forkability of glich + easier maintenance of a github repo. Is there any solution out there we can look at?

Reply:
If each example is it's own github repo, you can push to glitch git repo:
https://dev.to/elvezpablo/using-git-with-glitch-5832
You could also do it automatically with a GitHub Action https://github.com/marketplace/actions/glitch-project-sync
I personally do it the other way, from the glitch console I'm git pull to sync from the github repo:
https://github.com/networked-aframe/networked-aframe/blob/master/docs/release-instructions.md

With an alternate solution like codesandbox, you can start a project from a folder in a github repo, example this url:
https://codesandbox.io/s/github/LXSMNSYC/solid-headless/tree/main/examples/accordion-example
takes the code directly from github:
https://github.com/lxsmnsyc/solid-headless/tree/main/examples/accordion-example

Reply:
You can sync to glitch from a sub-path within a repo too.

I created [this one](https://glitch.com/edit/#!/pinto-generated-beast?path=index.html%3A9%3A49) by:
- In Glitch: Tools / Import & Export / Import from GithHUb
- Entering this path: aframevr/aframe:/examples/docs/aincraft
- After the code is loaded, modifying the A-Frame path to `https://aframe.io/releases/1.4.1/aframe.min.js`

See screenshot for steps...
![image](https://user-images.githubusercontent.com/16045703/213419616-27e1fb7b-2b3f-4cdb-a4e3-a71203b90c0c.png)

Reply:
For codesandbox.io, the following URL will autmatically clone from GitHub nto codesandbox.

https://githubbox.com/aframevr/aframe/tree/master/examples/docs/aincraft

To make it work, you need to fork the sandbox & modify the A-Frame path to https://aframe.io/releases/1.4.1/aframe.min.js

Reply:
Codesandbox is the lowest maintenance solution, just requires that we:
- update the examples to use a CDN URL for A-Frame, rather than using the local copy
- include the relevant githubbox URL in the documentation (or maybe in a corner overlay i the experience itself)

Suggest we also update the `npm run prerelease` command to update A-Frame version strings in HTML files in the `examples\docs` folder.
https://github.com/aframevr/aframe/pull/5213#discussion_r1069117087

I think that could be a one-liner [here](https://github.com/aframevr/aframe/blob/96706c46f1633e34816198e2258bf5befeece985/scripts/release.js#L34) like this:

```
glob.sync('examples/docs/**/*.html').forEach(updateDoc);
```

Reply:
Is moving to codesandbox OK, or is there an attachment to using glitch?

Reply:
I also found this documentation for github/glitch sync that seems to also promise a zero-maintenance position as well (after some initial setup).
https://github.com/javierarce/glitchub

Reply:
Proposal: let's get to a stable state with consistent up-to-date working code for all examples in examples/docs.

Then drive the sync to glitch and/or codesandbox as a separate issue/PR.

Now that #5215 is merged I'll update the remaining PRs to refer to examples/docs, which will get us to a consistent stable position, and we can move forwards from there.

Reply:
One more lead - https://glitch.new/
"Just type glitch.new in front of any GitHub repo URL to import it directly to Glitch, like [glitch.new/github.com/11ty/eleventy-base-blog](https://glitch.new/github.com/11ty/eleventy-base-blog)."

Unfortunately this only seems to work with a whole repo (aframevr/aframe), not a sub-part of that repo.  That would be a nice enhancement to get from glitch.

Reply:
I raised an enhancement request in glitch support forum.  
https://support.glitch.com/t/glitch-new-for-sub-trees-of-github-repos/59049

Reply:
glitch design looks cleaner and it's what the community is used to. It will take time to reeducate and there's so much stuff on glitch already. Not sure I understand the drawback with glitch if we can sync directly from the repo. 

About pointing `examples/docs` to a release. It will make harder to test locally those examples.

Reply:
Moving discussion to #5222 

Looks like a solution base on https://github.com/javierarce/glitchub is going to be the best way forwards.